### PR TITLE
commiting changes to todays branch from mitaka_21st_march_base_with_h…

### PR DESCRIPTION
**Pull request by AkashDeepSrivastava**
…andle_error_cases

### CODE REVIEW

File: gbpservice/nfp/orchestrator/modules/device_orchestrator.py [hunk 1]
```diff
@@ -810,13 +812,18 @@ def _post_configure_device_graph(self, nfp_context):
...
         nfp_context = event.data
-
+        nf_id = nfp_context['network_function']['id']
         # Get the results of PLUG_INTERFACES & PERFORM_HEALTH_CHECK events
         # results.
         results = event.graph.get_leaf_node_results(event)
 
         for result in results:
             if result.result.lower() != 'success':
+                if result.id == "PLUG_INTERFACES":
```
Comment (mak-454) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py]: make this internal event.

Comment (mak-454) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py]: call _create_event

File: gbpservice/nfp/orchestrator/modules/device_orchestrator.py [hunk 2]
```diff
@@ -1066,6 +1073,9 @@ def create_device_configuration(self, event):
                 'network_function_device': network_function_device}})
 
         if not config_params:
+            self._create_event(event_id='DRIVER_ERROR',
```
Comment (mak-454) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py:1071]: make this internal_event and use _create_event method

File: gbpservice/nfp/orchestrator/modules/device_orchestrator.py (hunk 1 above)
Comment (mak-454) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py]: Move this plug_interfaces_failed code to plug_interfaces() method in drivers/orchestration_driver.py

File: gbpservice/nfp/orchestrator/modules/device_orchestrator.py (hunk 2 above)
Comment (AkashDeepSrivastava) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py:1071]: this is already internal event

File: gbpservice/nfp/orchestrator/modules/device_orchestrator.py (hunk 1 above)
Comment (AkashDeepSrivastava) [inline gbpservice/nfp/orchestrator/modules/device_orchestrator.py]: Did the changes,need to test

**Pull request merged**